### PR TITLE
Add support for zwave-js 10.3

### PR DESCRIPTION
**Pull request by raman325**
We are mainly implementing https://github.com/zwave-js/node-zwave-js/pull/5121 and https://github.com/zwave-js/node-zwave-js/pull/5076

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -458,9 +458,9 @@ interface {
 }
 ```
 
-#### [Begin Firmware Update](https://zwave-js.github.io/node-zwave-js/#/api/node?id=beginfirmwareupdate)
+#### [Update Firmware](https://zwave-js.github.io/node-zwave-js/#/api/node?id=updatefirmware)
```
Comment (balloob) [inline README.md:461]: should we still docuemnt the old command?

File: src/lib/outgoing_message.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export type JSONValue =
   | number
   | boolean
   | null
+  | undefined
```
Comment (balloob) [inline src/lib/outgoing_message.ts:19]: JSON does not have `undefined`. In fact, it's the greatest hack in JavaScript history, as undefined keys are removed from the output:

```
> JSON.stringify({hello: undefined})
"{}" 
```

Comment (raman325) [reply src/lib/outgoing_message.ts:19]: yeah it was a typing issue with the result. I could use `as any` instead

File: README.md (hunk 1 above)
Comment (raman325) [reply README.md:461]: I don't think so. It's deprecated and going to get removed in a future release. I left it for backwards compatibility but it should not be used by new clients

**Pull request merged**